### DISCUSSION TITLE
True Pico 8 Support? Please please please 😃

### DISCUSSION THREAD

**Discussion by michaelmjh** [OnionUI/Onion#1250] [Ideas]
Fake 8 is just not reliable enough!

**Comment by XK9274** [comment]
This is the current state.. No promises either.

Runs like crap (1-2fps). spams console with joystick errors.

![script_000](https://github.com/OnionUI/Onion/assets/47260768/b919d0a1-6df4-41a7-a3dd-4a2521fc906f)
![script_001](https://github.com/OnionUI/Onion/assets/47260768/dc5db2b8-1504-465f-b5b2-0f7bebfe2d76)
![script_002](https://github.com/OnionUI/Onion/assets/47260768/83e3c704-526c-4e3d-a3c0-e5f4afa7c78a)
![script_003](https://github.com/OnionUI/Onion/assets/47260768/2311fff5-81e5-4489-a018-6faa54397991)

**Comment by XK9274** [comment]
![script_004](https://github.com/OnionUI/Onion/assets/47260768/9221c197-06bc-4c37-bd21-6b0bb71b25c3)
![script_005](https://github.com/OnionUI/Onion/assets/47260768/50a5b0e0-b8d2-447d-8268-43b214d24c15)
![script_006](https://github.com/OnionUI/Onion/assets/47260768/ae96a607-3530-49c2-8d93-decdc6ebeabf)

https://github.com/OnionUI/Onion/assets/47260768/8e5c6c00-0e46-4bd3-8415-8ca456ff1c90


ok maybe i lied

**Reply by michaelmjh** [reply]
Wow! You are a genius! If it could be added to Onion, I’ll be the happiest guy ever 😃
